http/client.go

```diff
@@ -37,8 +37,27 @@ type HttpClient interface {
 	PostForm(url string, data url.Values) (resp *http.Response, err error)
 }
 
+type Option func(*options)
```

Comment:
Perhaps a note on the independence of this type from the one in _client/client.go_